examples/MQ/serialization/sampler.cxx

```diff
@@ -32,7 +41,7 @@ class Ex2Sampler : public fair::mq::Device
     void Init() override
     {
         fFileName = fConfig->GetValue<std::string>("input-file");
-        fInputFile = TFile::Open(fFileName.c_str(), "READ");
+        fInputFile = std::make_unique<TFile>(fFileName.c_str(), "READ");
```

Comment:
```suggestion
        fInputFile.reset(TFile::Open(fFileName.c_str(), "READ"));
```

I think, TFile::Open is recommended over instantiating the TFile directly, because TFile::Open could return a specialized subclass (like for network access).

Reply:
Didn't know about that, I'll apply your suggestion then.